### DISCUSSION TITLE
Auto-promotion for older freights?

### DISCUSSION THREAD

**Discussion by acanizares** [akuity/kargo#3013] [Q&A] [answered]
I have an example project with 3 stages, all of them with auto-promotion enabled.
I noticed that when manually promoting an older freight to stage 1, it is not auto-promoted to stages 2 and 3. Is this expected?

Even though that old freight was previously verified in all stages during the forward deployment, the verification might look different during a rollback. For instance, the current database schemas are not compatible with that older code. For that reason I was hoping to be able to promote an old freight automatically through the whole pipeline and to run some verifications before promoting to production.

Thanks!

**Comment by krancour** [comment, marked as answer]
This is the expected behavior. If auto-promotion is enabled on a Stage, it is only the newest Freight available to that Stage that is eligible.
